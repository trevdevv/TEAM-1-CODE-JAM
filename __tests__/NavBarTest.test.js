import React from "react";
import "babel-polyfill";
import renderer from "react-test-renderer";
import {
  Route,
  Switch,
  BrowserRouter as Router,
  withRouter,
} from "react-router-dom";

// Import side nav component
import NavBar from "../src/components/NavBar";

// Initialize chrome
const chrome = require("sinon-chrome");
window.chrome = chrome;

// Test side nav setup
test("Testing navigation bar", () => {
  const component = renderer.create(
    <Router>
      <NavBar />
    </Router>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
