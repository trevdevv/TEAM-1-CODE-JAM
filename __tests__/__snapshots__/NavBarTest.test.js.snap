// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing navigation bar 1`] = `
<div>
  <nav
    id="nav-background"
  >
    <div
      className="nav-wrapper"
    >
      <a
        className="brand-logo"
        href="#"
      >
        Team 1
      </a>
      <ul
        className="right"
      >
        <li>
          <a
            href="#"
          >
            About
          </a>
        </li>
      </ul>
    </div>
  </nav>
</div>
`;
